### DISCUSSION TITLE
How does bridging RSS to Bluesky work?

### DISCUSSION THREAD

**Discussion by venkr** [snarfed/bridgy-fed#1420] [Q&A]
I'd love to follow a website, say any substack-based website, like eg:
`https://www.astralcodexten.com`
They have their RSS feed at a pretty common feed URL:
`https://www.astralcodexten.com/feed`

However, when I try bridging (ie: https://fed.brid.gy/web/astralcodexten.com)
- The "activity" section seems kinda broken, see screenshot
<img width="876" alt="image" src="https://github.com/user-attachments/assets/ac147a58-69ba-4425-8717-b4ca7e66c83c">
- I see nothing under "Feed"

I'm partially interested in this because I'd love to bridge my own blog to BlueSky as well, and I'm a little confused about how to set that up!

**Comment by venkr** [comment]
Okay, just tested this flow with my own website and it fails:
- It has `<link rel="alternate" type="application/rss+xml" href="/rss" />` on the homepage
- It has the rss feed available on `/rss` (and `/rss.xml`, `/feed`, `/feed.xml` just to be sure)
- The RSS feed works correctly with RSS Readers (tested via: feeder.co)
- I can see the `Bridgy Fed (https://fed.brid.gy/)` crawler hit my homepage (but not my rss feed URL)

Here's the URL: https://fed.brid.gy/web/venki.dev

Open to any tips! Ty :)

**Comment by snarfed** [comment]
Hey, sorry for the trouble! Not sure what's going on here. RSS/Atom feeds are working in general, eg https://bsky.app/profile/nature.com.web.brid.gy , so something must be going on with your site and Scott's. I'll look.

**Comment by snarfed** [comment]
So right now https://venki.dev/ redirects to https://venki.dev/notes , which Bridgy Fed doesn't support. It only tries to bridge full web sites, which it determines by working with the home page, ie path `/`.

In your case, redirecting `/` to `/notes` does show that you own the site, as opposed to just being a user. I'm happy to take this as a feature request to support that if you want!

**Reply by venkr** [reply]
Thanks for the super quick response!

Just changed `https://venki.dev` to not redirect anymore - that makes a little more sense anyways. 
But it looks like it's still behaving oddly - I see a `... updated their profile` message, but I see no feed and don't see a bluesky account created.

I also don't seem to see a pageview on my RSS file, so perhaps I'm linking to it wrong 🤔

Let me know if you have any tips about what else could be wrong :)

**Reply by snarfed** [reply]
We're not as good as we should be at recovering web sites that hit a problem like this in the middle of setup. That's https://github.com/snarfed/bridgy-fed/issues/1202 . I'll see if I can nudge yours manually soon!

**Reply by snarfed** [reply]
Thanks for your patience here! Your bridged account in Bluesky looks ok now: https://bsky.app/profile/venki.dev.web.brid.gy

**Comment by snarfed** [comment]
astralcodexten.com was wedged in a weird way, I've kicked it and it's posting now, https://bsky.app/profile/astralcodexten.com.web.brid.gy . Its profile details are unhappy for a different reason, I'll look into that.

**Reply by venkr** [reply]
Awesome - thanks!
